susemanager/src/mgr_bootstrap_data.py

```diff
@@ -152,42 +108,7 @@
 ]
 
 RES7 = [
-    "iproute",
-    "iptables",
-    "libmnl",
-    "libnetfilter_conntrack",
-    "libnfnetlink",
     "logrotate",
```

Comment:
Stricly speaking, I guess we could get rid of logrotate, as it's present by default in Amazon Linux 2, and the most minimalistic installation I could do for the latest CentOS7.

But since it's here, and it's indeed a bundle dependency, let's keep it.